client/dashboard/dashboard-charts/config.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import { charts as ordersCharts } from '../../analytics/report/orders/config';
+import { charts as productsCharts } from '../../analytics/report/products/config';
+import { charts as revenueCharts } from '../../analytics/report/revenue/config';
```

Comment:
I think for these imports you can just do `analytics/report/revenue/config`, etc.